include/bluetooth/services/ras.h

```diff
@@ -130,6 +131,61 @@ struct ras_subevent_header {
 } __packed;
 BUILD_ASSERT(sizeof(struct ras_subevent_header) == BT_RAS_SUBEVENT_HEADER_LEN);
 
+typedef void (*bt_ras_rreq_rd_ready_cb_t)(struct bt_conn *conn, uint16_t ranging_counter);
+typedef void (*bt_ras_rreq_rd_overwritten_cb_t)(struct bt_conn *conn, uint16_t ranging_counter);
+typedef void (*bt_ras_rreq_ranging_data_get_complete_t)(int err, uint16_t ranging_counter);
+
+/** @brief Allocate a RREQ context and assign GATT handles. Takes a reference to the connection.
+ *
+ * @param[in] dm   Discovery Object.
+ * @param[in] conn Connection Object.
+ *
+ * @retval 0 If the operation was successful.
+ *           Otherwise, a negative error code is returned.
+ */
+int bt_ras_rreq_alloc_and_assign_handles(struct bt_gatt_dm *dm, struct bt_conn *conn);
+
+/** @brief Get ranging data for given ranging counter.
+ *
+ * @note This should only be called after receiving a ranging data ready notification/indication and
```

Comment:
Maybe this is more like what it'll look like to the app?
```suggestion
 * @note This should only be called after receiving a ranging data ready callback and
```